openmc/mgxs/library.py

```diff
@@ -13,6 +15,11 @@
 if sys.version_info[0] >= 3:
     basestring = str
 
+# The following represent the most accurate MGXS generation strategy
+# for use in the MG mode of OpenMC.
+OPENMC_MG_MGXS_TYPES = ['transport', 'absorption', 'nu-fission', 'chi',
```

Comment:
I actually think I may remove this altogether now.  With your PR #644, the proper set would be to use `['total 'absorption', 'nu-fission', 'chi', 'scatter matrix', 'nu-scatter matrix']` and then I'd have to force the user to change the default correction and the default scattering order.  So in the end having this doesn't save the user too much pain in setting up the MGXS for OpenMCs MG mode.
Plus, on top of all that, I started thinking yesterday that maybe its better to force the users to think more about how they are making their MGXS since using MC to generate MGXS is relatively new and because I dont like having MGXS generation being a black box.

So... I've now removed this line.